### ISSUE TITLE
Only stderr is printed on error

### ISSUE THREAD

**Issue by oschwald** [houseabsolute/precious#26]
The docs state:

> If the command fails somehow, precious will print out stdout (and stderr) output.

However, it appears that only stderr is printed.

For instance, given this `.precious.toml` file:

```toml
[commands.test]
type = "lint"
cmd = [ "./test.sh" ]
ok_exit_codes = 0
include = [ "**/*" ]
```

And this `test.sh` in the same directory:

```sh
#!/bin/bash

echo "STDOUT"
echo "STDERR" 1>&2
exit 1
```

I see the following:

```
$ precious lint --all
💍 Linting all files in the project
💥 error test: .precious.toml
💥 error test: test.sh
Errors when linting files:
  ▶ .precious.toml [commands.test]
    Failed to execute command `./test.sh .precious.toml`: Got unexpected exit code 1 from `./test.sh .precious.toml`. Stderr was STDERR

  ▶ test.sh [commands.test]
    Failed to execute command `./test.sh test.sh`: Got unexpected exit code 1 from `./test.sh test.sh`. Stderr was STDERR
```

**Comment by oalders** [comment]
I think it needs something like:

```diff
diff --git a/src/command.rs b/src/command.rs
index 16c0b64..8c1c673 100644
--- a/src/command.rs
+++ b/src/command.rs
@@ -20,11 +20,12 @@ pub enum CommandError {
     #[error("Got unexpected exit code {code:} from `{cmd:}`")]
     UnexpectedExitCode { cmd: String, code: i32 },

-    #[error("Got unexpected exit code {code:} from `{cmd:}`. Stderr was {stderr:}")]
+    #[error("Got unexpected exit code {code:} from `{cmd:}`. Stderr was {stderr:}. Stdout was {stdout:}")]
     UnexpectedExitCodeWithStderr {
         cmd: String,
         code: i32,
         stderr: String,
+        stdout: String,
     },

     #[error("Ran `{cmd:}` and it was killed by signal {signal:}")]
@@ -136,6 +137,7 @@ fn output_from_command(
                         cmd: cstr,
                         code,
                         stderr: String::from_utf8(output.stderr)?,
+                        stdout: String::from_utf8(output.stdout)?,
                     }
                     .into());
                 }
```

**Comment by autarch** [comment]
Thanks for the bug report! This is fixed in v0.1.5.
